dom.bs

```diff
@@ -993,9 +995,9 @@ for historical reasons.
 <h3 id=interface-eventtarget>Interface {{EventTarget}}</h3>
 
 <pre class=idl>
-[Constructor,
- Exposed=(Window,Worker,AudioWorklet)]
+[Exposed=(Window,Worker,AudioWorklet)]
 interface EventTarget {
+  constructor();
```

Comment:
Newline?

Reply:
I tried to be consistent with what was previously done within each interface. Where methods and attributes were already separated by a blank line I added a newline, and where there were no blank lines I didn't add one.

Is there a rule I can apply to all of the affected interfaces?

Reply:
I was thinking we'd add a newline after each constructor (unless there's nothing following it). The other newlines don't have a good rule I think. They're mostly added for some subjective measure of readability.